### PR TITLE
Added GCE instructions, changing from Amazon Linux to Ubuntu

### PR DESCRIPTION
**Pull request by arun-gupta**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,13 +61,15 @@ firectl \
 Getting Started on AWS
 ---
 
-- Create an `i3.metal` instance using Amazon Linux 2
+- Create an `i3.metal` instance using Ubuntu 18.xx
```
Comment (samuelkarp) [inline README.md]: We should specify more precisely which release should be used.  Also, what's the reason for switching from Amazon Linux 2 to Ubuntu.

File: gce.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
+= Getting Started with Firecracker on Google Compute Engine
```
[the remark this answers is not in the document]
Comment (arun-gupta) [reply gce.adoc]: Yes, this file is no longer needed anyway and have been removed

File: README.md [hunk 3]
```diff
@@ -61,13 +61,15 @@ firectl \
...
 - Build latest version of firectl:
 
   ```
-  sudo yum install -y git
+  sudo apt-get -y update
+  sudo apt-get -y upgrade
+  sudo apt-get install -y git make build-essential
+  sudo snap install --classic go
```
[the remark this answers is not in the document]
Comment (arun-gupta) [reply README.md]: Updated to specifically install Go 1.11

File: README.md (hunk 1 above)
Comment (arun-gupta) [reply README.md]: This was after a discussion with @nmeyerhans a couple of weeks back. Also, Ubuntu is a more likely O/S to be chosen by developers, particularly if we're going across multiple clouds.

File: README.md [hunk 4]
```diff
@@ -58,16 +58,23 @@ firectl \
...
-Getting Started on AWS
+Getting Started with Firectl
 ---
 
-- Create an `i3.metal` instance using Amazon Linux 2
+- Set up your machine as explained at https://github.com/firecracker-microvm/firecracker/blob/master/docs/dev-machine-setup.md
+- Install Go 1.11:
+
+  ```
+  sudo add-apt-repository ppa:longsleep/golang-backports
```
Comment (samuelkarp) [inline README.md]: Who maintains this PPA?  I think it'd be better to suggest installation from https://golang.org/dl/ given that we know those binaries are produced by the upstream project.

**Comment by samuelkarp** [comment]
@arun-gupta Do you plan to address my question above, or should we close this for now?

**Comment by arun-gupta** [comment]
close for now

**Comment by samuelkarp** [comment]
Thanks @arun-gupta!

**Pull request closed without merging**